src/assets/scripts/math/flightMath.js

```diff
@@ -155,3 +160,14 @@ export const dist_to_boundary = (airport, pos) => {
     // TODO: hmm, `position.position`? that seems fishy
     return abs(distance2d(pos, airport.position.position) - airport.ctr_radius);
 };
+
+/**
+ *
+ *
+ *
+ * @function calculateHeadingFromTwoPositions
+ * @param positionEnd {array}
+ * @param positionStart {array}
+ * @return {number}
+ */
+export const calculateHeadingFromTwoPositions = (positionEnd, positionStart) => vradial(vsub(positionEnd, positionStart));
```

Comment:
This would probably make a lot more sense with the arguments reversed. I would tend to want to invoke this function as `calculateHeadingFromTwoPositions(aircraft.position, another.position);`, and would expect the heading FROM `aircraft` TO the other position, but here I would get the reverse. With `vsub()` it would make sense to feed parameters as "final minus initial", but with the more user-friendly named function, I definitely think the reverse is more intuitive.